Lib/imaplib.py

```diff
@@ -807,7 +807,6 @@ def proxyauth(self, user):
         (typ, [data]) = <instance>.proxyauth(user)
         """
 
-        name = 'PROXYAUTH'
         return self._simple_command('PROXYAUTH', user)
```

Comment:
Stylistically the better change here would be
```
        name = 'PROXYAUTH'
        return self._simple_command(name, user)
```

That matches the style in the rest of the file.  It's not a big deal, though.